docs/cluster-agent/HORIZONTAL_POD_AUTOSCALING.md

```diff
@@ -0,0 +1,262 @@
+# Datadog Cluster Agent (DCA) | Getting Started
+
+The DCA is a **beta** feature, if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help)
+
+## Introduction
+
+The Horizontal Pod Autoscaling feature has been introduced in [Kubernetes v1.2](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#before-you-begin).
+It allowed users to autoscale off of basic metrics like CPU, but required a resource called the metrics-server to run along side your application.
+As of Kubernetes v1.6, it is possible to autoscale off of [custom metrics](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics).
+Custom metrics are user defined and are collected from within the cluster.
+As of Kubernetes v1.10, support for external metrics was introduced so users can autoscale off of any metric from outside the cluster that is collected for you by Datadog.
+
+The custom and external metric provider as opposed to the metrics server, are resources that have to be implemented and registered by the user.
+
+As of v1.0.0, the DCA implements the External Metrics Provider interface.
+This walkthrough will explain how to set it up and how it can help you autoscale your Kubernetes workload based off of your Datadog metrics.
+
+
+## Requirements
+
+1. Have Kubernetes v1.10, this is so the External Metrics Provider resource can be registered against the API Server.
+2. You will need to have the Aggregation layer enabled as well, please refer to the official guide:
+https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/
+
+## Walkthrough
+
+### Preliminary disclaimer
+
+Autoscaling over External Custom Metrics does not require the node agent to be running. Nevertheless, for the sake of this walkthrough, we will autoscale an nginx deployment based off of nginx metrics, collected by a node agent.
+To this extent, should you want to further proceed, we are assuming that:
```

Comment:
This sentence is a bit too complex. Let's change it to `If you want to proceed further, we are assuming that:`